<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageConnectionWidget</class>
 <widget class="QWidget" name="ManageConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>262</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabWired">
      <attribute name="title">
       <string>Wired</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTreeWidget" name="listWired">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>210</height>
          </size>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Connection</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last Used</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="AddEditDeleteButtonSet" name="buttonSetWired" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWireless">
      <attribute name="title">
       <string>Wireless</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTreeWidget" name="listWireless">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Connection</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last Used</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="AddEditDeleteButtonSet" name="buttonSetWireless" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCellular">
      <attribute name="title">
       <string>Mobile Broadband</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTreeWidget" name="listCellular">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Connection</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last Used</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="AddEditDeleteButtonSet" name="buttonSetCellular" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabVpn">
      <attribute name="title">
       <string>VPN</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTreeWidget" name="listVpn">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Connection</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last Used</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="AddEditDeleteButtonSet" name="buttonSetVpn" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPppoe">
      <attribute name="title">
       <string>DSL</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTreeWidget" name="listPppoe">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Connection</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last Used</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="AddEditDeleteButtonSet" name="buttonSetPppoe" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AddEditDeleteButtonSet</class>
   <extends>QWidget</extends>
   <header>addeditdeletebuttonset.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
